apps/web/src/components/editor/tiptap.tsx

```diff
@@ -443,14 +444,36 @@ function TiptapWrapper(
     }
   }, [isHydrating]);
 
+  useEffect(() => {
+    if (!editorContainerRef.current) return;
+
+    const handleWheel = (e: WheelEvent) => {
+      if (e.ctrlKey) {
+        e.preventDefault();
+        const delta = e.deltaY;
+        let zoom = editorConfig.zoom;
+        zoom += Math.round(delta);
+        zoom = Math.min(EDITOR_ZOOM.MAX, Math.max(EDITOR_ZOOM.MIN, zoom));
+        setEditorConfig({ zoom });
```

Comment:
I think this can be shortened to something like:
```suggestion
        const zoom = Math.min(EDITOR_ZOOM.MAX, Math.max(EDITOR_ZOOM.MIN, editorConfig.zoom + Math.round(delta));
        setEditorConfig({ zoom });
```